README.md

```diff
@@ -3,27 +3,42 @@
 ## Prerequisites
 - [Numaflow](https://numaflow.numaproj.io/quick-start/#installation)
 - [Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/)
-
+- [AWS](https://aws.amazon.com/)
 
 ## Instructions
-1. Fork and clone the repository to your local and build numalogic-prometheus docker image.
+
+- Fork and clone the repository to your local and build numalogic-prometheus docker image.
 
 ```shell
 git@github.com:numaproj/numalogic-prometheus.git
 
 docker build -t numalogic-prometheus . && k3d image import docker.io/library/numalogic-prometheus
 ```
 
-2. Create numalogic-prometheus namespace.
+- Create numalogic-prometheus namespace.
 ```shell
 kubectl create namespace numalogic-prometheus
 
 kubectl config set-context --current --namespace=numalogic-prometheus
 ```
 
-3. MLflow S3 bucket setup
-- Create an [AWS S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html), called `mlflow-nuamlogic-prometheus`
-- Create an [AWS IAM Role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html), with the following policy.
+
+## Redis Setup
+Install redis in `numalogic-prometheus` namespace and copy the password to notepad.
+
+```shell
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm install numalogic bitnami/redis-cluster
+echo $(kubectl get secret --namespace "numalogic-prometheus" numalogic-redis-cluster -o jsonpath="{.data.redis-password}" | base64 -d)
+```
+
+## MLflow Setup
+
+### AWS S3 Bucket Setup:
+- Create an [AWS S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html)
+- Create an [AWS IAM Role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html), with the following policy. 
+ 
```

Comment:
@nkoppisetty can we tackle https://github.com/numaproj/numalogic-prometheus/issues/9 next?